go/vt/workflow/resharding/parallel_runner.go

```diff
@@ -63,13 +62,13 @@ type ParallelRunner struct {
 // NewParallelRunner returns a new ParallelRunner.
 func NewParallelRunner(ctx context.Context, rootUINode *workflow.Node, cp *CheckpointWriter, tasks []*workflowpb.Task, executeFunc func(context.Context, *workflowpb.Task) error, concurrencyLevel level, enableApprovals bool) *ParallelRunner {
 	if len(tasks) < 1 {
-		log.Fatal(errors.New("BUG: No tasks passed into ParallelRunner"))
+		log.Fatalf("BUG: No tasks passed into ParallelRunner")
```

Comment:
Please use `log.Fatal` (without f) if the parameter should not be interpreted as format string. This mostly important if you log the content of a variable e.g. `log.Fatal(err)`. But to be consistent, let's do this for string literals as well.